schema.sql

```diff
@@ -6,5 +6,23 @@ CREATE TABLE
         escape_attempts SMALLINT NOT NULL,
         neutered BOOLEAN NOT NULL,
         weight_kg NUMERIC(10, 2) NOT NULL,
-        species VARCHAR(50) NULL
+        species_id INT,
+        owner_id INT,
+        CONSTRAINT fk_species FOREIGN KEY (species_id) REFERENCES species (id) ON DELETE CASCADE,
+        CONSTRAINT fk_owners FOREIGN KEY (owner_id) REFERENCES owners (id) ON DELETE CASCADE
+    );
```

Comment:
- [ ] When implementing modifications to existing database tables, it is considered a recommended practice to utilize the ALTER keyword for executing new statements that enact these changes. In this context:

- The "species" column is being removed.
- A new column named "species_id" is being introduced, establishing a foreign key relationship with the "species" table.
- An additional column named "owner_id" is being appended, establishing a foreign key relationship with the "owners" table.

The SQL commands for these actions can be structured as follows:

```sql
ALTER TABLE animals DROP COLUMN species;
ALTER TABLE animals ADD COLUMN species_id INT REFERENCES species(id);
ALTER TABLE animals ADD COLUMN owner_id INT REFERENCES owners(id);
```

This approach ensures that alterations to the database schema are carried out in a controlled manner, aligning with best practices for maintaining data integrity and database structure.

Please remove the commands from lines 9 to 12 for the animal table and use the above-suggested method.

Reply:
Hi @bobb-Rob I hope you are well and cheers for reviewing this pr! ⭐️

---

All changes were executed using the **psql** tool:

I utilized the ALTER command to appropriately modify tables for deletion and updating in accordance with the new requirements.

These SQL statements were implemented via **psql** to address the subsequent requirements.

However, the clarity regarding whether these statements should be documented or if direct modifications to the target table were necessary was not provided by the requirements or were ambiguous.

If you require additional details or have any inquiries, please don't hesitate to let me know. 🤠

Reply:
* As mentioned in the [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=Remove%20column%20species) you are to remove the species column, currently, your queries hasn't solved this yet. kindly fix this

Reply:
Hi Guys! Good day 🌞

All right then, changes will be made to follow the suggested required changes 🎯 

Cheers again! 😃